.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: 🎉 Feature Request
+about: You have a neat idea that should be implemented? 🎩
+---
+
+### Feature Request
+
+#### Why
+
+<!--
+Explain why you believe this feature should be implemented.
+-->
+
+#### What
+
+<!--
+Describe the behavior you would like to see.
+-->
```

Comment:
I suggest flipping the what and why; often the "why" becomes quite apparent when describing the suggested feature. I'm also blindly assuming that most such requests come from "I want to use feature X that isn't supported" thoughts, rather than "I have problem X that needs to be solved by a new feature". In that case, the thought process would always lead someone to a "what" before going on to the "why".

```suggestion
#### What

<!--
Describe the behavior you would like to see.
-->

#### Why

<!--
Explain why you believe this feature should be implemented.
-->
```